TensorFlow is failing with Bazel@HEAD

https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/2997#0187e4ef-95bb-43c3-84ea-7cd94dbf0b7d

Platforms : Linux and Windows

Logs :  
```
ERROR: /var/lib/buildkite-agent/builds/bk-docker-5xrg/bazel-downstream-projects/tensorflow/tensorflow/cc/BUILD:766:22: Linking tensorflow/cc/ops/functional_ops_gen_cc [for tool] failed: (Exit 1): gcc failed: error executing command (from target //tensorflow/cc:ops/functional_ops_gen_cc)
  (cd /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/c5e474199d1c61ddf0091ab0c5156762/execroot/org_tensorflow && \
  exec env - \
    PATH=/var/lib/buildkite-agent/.cache/bazelisk/local/-tmp-tmpjryk49s9-bazel/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
    PWD=/proc/self/cwd \
    ZERO_AR_DATE=1 \
  /usr/bin/gcc @bazel-out/k8-opt-exec-ST-86d428d70b9d/bin/tensorflow/cc/ops/functional_ops_gen_cc-2.params)
# Configuration: 480f85a7755fc62a440f8fbe1f9f17e1ba62abf41ed236d08b174eb7e701c839
# Execution platform: @local_execution_config_platform//:platform
bazel-out/k8-opt-exec-ST-86d428d70b9d/bin/_solib_k8/_U_S_Stensorflow_Scc_Cops_Sfunctional_Uops_Ugen_Ucc___Utensorflow/libtensorflow_framework.so.2: error: undefined reference to 'tsl::table::NewLRUCache(unsigned long)'
bazel-out/k8-opt-exec-ST-86d428d70b9d/bin/_solib_k8/_U_S_Stensorflow_Scc_Cops_Sfunctional_Uops_Ugen_Ucc___Utensorflow/libtensorflow_framework.so.2: error: undefined reference to 'tensorflow::GpuIdManager::TfToPlatformDeviceId(tsl::gtl::IntType<tsl::TfDeviceId_tag_, int>, tsl::gtl::IntType<tsl::PlatformDeviceId_tag_, int>*)'
collect2: error: ld returned 1 exit status
Target //tensorflow/tools/pip_package:build_pip_package failed to build
(04:36:51) INFO: Elapsed time: 1816.483s, 
```
Steps followed :  
```
1. git clone -v https://github.com/tensorflow/tensorflow.git
2. git reset cf9014ebbc63ae2e905930e333bde676b87f0167 --hard
3. export USE_BAZEL_VERSION=8087520b153832656695fef6f3654ed432642c98
4. bazel build //tensorflow/tools/pip_package:build_pip_package

```
CC @meteorcloudy @oquenchil

--- a/.bazelrc
+++ b/.bazelrc
@@ -122,9 +122,10 @@ build --define=grpc_no_ares=true
 # hope to provide a better mechanism for control over symbol exporting, and
 # then tackle this issue again.
 #
-# TODO: Remove this line once TF doesn't depend on Bazel wrapping all library
-# archives in -whole_archive -no_whole_archive.
+# TODO: Remove the following two lines once TF doesn't depend on Bazel wrapping
+# all library archives in -whole_archive -no_whole_archive.
 build --noincompatible_remove_legacy_whole_archive
+build --features=-force_no_whole_archive
 
 # TODO(mihaimaruseac): Document this option or remove if no longer needed
 build --enable_platform_specific_config

